content/200-concepts/050-overview/300-prisma-in-your-stack/03-is-prisma-an-orm.mdx

```diff
@@ -292,68 +307,138 @@ generator client {
+  title     String
+  content   String? @map("post_content")
+  published Boolean @default(false)
+  author    User?   @relation(fields: [authorId], references: [id])
+  authorId  Int?
+}
+
 model User {
-  id        Int      @id
-  birthDate DateTime
-  email     String   @unique
-  firstName String?
-  lastName  String?
+  id    Int     @id @default(autoincrement())
+  email String  @unique
+  name  String?
+  posts Post[]
 }
 ```
 
-2. `prisma migrate save --experimental`: Save the migration. Typically migrations are saved in the code repository
-3. `prisma migrate up --experimental`: Run the migration which will create the database schema
-4. `prisma generate`: Re-generate Prisma Client
+2. Run `prisma migrate` to generates the SQL for the migration, apply it to the database, and generate Prisma Client.
 
 For any further changes to the database schema:
 
-1. Apply changes to the Prisma schema, e.g. add a `registrationDate` field to the `User` model
-1. Follow steps 2-4.
+1. Apply changes to the Prisma schema, e.g., add a `registrationDate` field to the `User` model
+1. Run `prisma migrate` again.
 
-The last two steps demonstrate how declarative migrations work by adding a field to the Prisma schema and using Prisma Migrate to transform the database schema to the desired state. Lastly, the Prisma Client was regenerated so that it reflects the updated schema.
+The last step demonstrates how declarative migrations work by adding a field to the Prisma schema and using Prisma Migrate to transform the database schema to the desired state. After the migration is run, Prisma Client is automatically regenerated so that it reflects the updated schema.
 
 If you don't want to use Prisma Migrate but still want to use the type-safe generated Prisma Client in a new project, see the next section.
 
 ##### Alternative for new projects without Prisma Migrate
 
-The workflow above relies on Prisma Migrate, which is still experimental. It is possible to use Prisma Client in a new project with a third-party migration tool instead of Prisma Migrate. For example, a new project could choose to use the Node.js migration framework [db-migrate](https://github.com/db-migrate/node-db-migrate) to create the database schema and migrations and Prisma Client for querying. In essence, this is covered by the [workflow for existing databases](#workflow-for-incremental-adoption-in-projects-with-an-existing-database).
+The workflow above relies on Prisma Migrate, which is still in early access. It is possible to use Prisma Client in a new project with a third-party migration tool instead of Prisma Migrate. For example, a new project could choose to use the Node.js migration framework [db-migrate](https://github.com/db-migrate/node-db-migrate) to create the database schema and migrations and Prisma Client for querying. In essence, this is covered by the [workflow for existing databases](#workflow-for-incremental-adoption-in-projects-with-an-existing-database).
```

Comment:
Is Prisma Migrate really already in "early access" or can this only be merged after the old experimental Migrate is gone?

Reply:
I'll add a link to the public documentation for the early access version.